### PR TITLE
Fix for: gtest pkg-config file contains wrong include path

### LINKED ISSUE #100834: gtest pkg-config file contains wrong include path
**Describe the bug**

The `includedir` in `gtest.pc` contains the store path twice:

```
includedir=/nix/store/0kcdjakdp3mfvgvvzls5si34x0rz6z8k-gtest-1.10.0-dev//nix/store/0kcdjakdp3mfvgvvzls5si34x0rz6z8k-gtest-1.10.0-dev/include
```

**To Reproduce**

```
$ cat $(nix-build --expr 'with import (builtins.fetchTarball "https://github.com/nixos/nixpkgs/archive/24c9b05ac53e422f1af81a156f1fd58499eb27fb.tar.gz") {}; gtest.dev')/lib/pkgconfig/gtest.pc
prefix=${pcfiledir}/../..
libdir=${prefix}//nix/store/vbfv5wqhizxlqpsm3p07xjh7fpl371i1-gtest-1.10.0/lib
includedir=/nix/store/0kcdjakdp3mfvgvvzls5si34x0rz6z8k-gtest-1.10.0-dev//nix/store/0kcdjakdp3mfvgvvzls5si34x0rz6z8k-gtest-1.10.0-dev/include

Name: gtest
Description: GoogleTest (without main() function)
Version: 1.10.0
URL: https://github.com/google/googletest
Libs: -L${libdir} -lgtest -lpthread
Cflags: -I${includedir} -DGTEST_HAS_PTHREAD=1 -lpthread
```

**Notify maintainers**

@Zoomulator

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.116, NixOS, 19.09.2520.289466dd6a1 (Loris)`
 - multi-user?: `yes`
 - sandbox: `relaxed`
 - version: `nix-env (Nix) 3.0pre20200829_f156513`

### PR DESCRIPTION
**Pull request by AtnNn**
###### Motivation for this change

Fixes https://github.com/NixOS/nixpkgs/issues/100834

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/gtest/default.nix [hunk 1]
```diff
@@ -15,6 +15,7 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./fix-cmake-config-includedir.patch
+    ./fix-pkgconfig-paths.patch
```
Comment (jtojnar) [inline pkgs/development/libraries/gtest/default.nix:18]: Is it possible to fetchpatch it?

Comment (AtnNn) [reply pkgs/development/libraries/gtest/default.nix]: I've changed it to use `fetchpatch`.

**Comment by zowoq** [comment]
@ofborg eval

**Comment by jtojnar** [comment]
Thanks.

**Pull request merged**